smithy-typescript-ssdk-libs/server-common/src/httpbinding/mux.spec.ts

```diff
@@ -71,8 +76,9 @@ describe("simple matching", () => {
       new HttpRequest({ method: "GET", path: "/greedy/a/b/c/d", query: { abc: "def" } }),
     ],
     "Test#MiddleGreedy": [
-      new HttpRequest({ method: "GET", path: "/mg/a/z" }),
-      new HttpRequest({ method: "GET", path: "/mg/a/b/c/d/z", query: { abc: "def" } }),
+      new HttpRequest({ method: "GET", path: "/mg/a/y/z" }),
+      new HttpRequest({ method: "GET", path: "/mg/a/b/c/d/y/z", query: { abc: "def" } }),
+      new HttpRequest({ method: "GET", path: "/mg/a/b/y/c/d/y/z", query: { abc: "def" } }),
```

Comment:
we should add a case like `"/mg/a/b/y/z/d/y/z"` too.

Do we have any test that validates that is captured in the label?

Reply:
`mux` doesn't do any capturing of label values - it just does path matching.

Reply:
You're right about the test case though, I added it